src/ResourceManager/KeyVault/Commands.KeyVault/Models/KeyVaultDataServiceClient.cs

```diff
@@ -40,9 +43,7 @@ public KeyVaultDataServiceClient(IAuthenticationFactory authFactory, AzureContex
                 throw new ArgumentNullException("httpClient");
 
             var credential = new DataServiceCredential(authFactory, context, AzureEnvironment.Endpoint.AzureKeyVaultServiceEndpointResourceId);
-            this.keyVaultClient = new KeyVaultClient(
-                credential.OnAuthentication,
-                httpClient);
+            this.keyVaultClient = new KeyVaultClient(credential.OnAuthentication);
```

Comment:
How is the httpClient used? Is it safe not to have it or add it as DelegatingHandler?